### PR TITLE
feat: Update 941160.ra

### PR DESCRIPTION
**Pull request by Xhoenix**
Added events from PortSwigger XSS CheatSheet
https://portswigger.net/web-security/cross-site-scripting/cheat-sheet

### CODE REVIEW

**Comment by RedXanadu** [comment]
@Xhoenix There have been some problems this afternoon which broke all recent PRs. The other PRs were okay after merging the fixed v4.0/dev branch into the PRs, but for some reason that has not fixed _this_ PR.

**Comment by theseion** [comment]
@Xhoenix please run `crs-toolchain regex update -a` to update the rules from the assembly files.

**Comment by Xhoenix** [comment]
@RedXanadu @theseion fixed

File: rules/REQUEST-941-APPLICATION-ATTACK-XSS.conf [hunk 1]
```diff
@@ -838,8 +838,8 @@ SecRule REQUEST_COOKIES|!REQUEST_COOKIES:/__utm/|REQUEST_COOKIES_NAMES|ARGS_NAME
     msg:'Node-Validator Deny List Keywords',\
     logdata:'Matched Data: %{TX.0} found within %{MATCHED_VAR_NAME}: %{MATCHED_VAR}',\
     tag:'application-multi',\
-    tag:'language-multi',\
     tag:'platform-multi',\
+    tag:'language-multi',\
```
Comment (theseion) [inline rules/REQUEST-941-APPLICATION-ATTACK-XSS.conf:842]: Why?

Comment (Xhoenix) [reply rules/REQUEST-941-APPLICATION-ATTACK-XSS.conf:842]: Idk. I didn't change it. I just did 'regex update' and push.

Comment (theseion) [reply rules/REQUEST-941-APPLICATION-ATTACK-XSS.conf:842]: That's weird... Nothing but the regex should be touched. Please revert this change.

Comment (theseion) [reply rules/REQUEST-941-APPLICATION-ATTACK-XSS.conf:842]: Please revert the change in tags.

**Review by theseion — CHANGES_REQUESTED** [review]

**Comment by dune73** [comment]
For the record, this is an update to a keyword list that was supposed to be automated in #2701 based on a known source for the keywords. Somehow this was skipped, maybe because the overhaul was very early.

Either way, we will need to get this done sooner or later.

The PR by @Xhoenix is very welcome.

File: rules/REQUEST-941-APPLICATION-ATTACK-XSS.conf (hunk 1 above)
Comment (Xhoenix) [reply rules/REQUEST-941-APPLICATION-ATTACK-XSS.conf:842]: I already did. For some reason it's not showing here. Try merging.

**Comment by theseion** [comment]
@Xhoenix I made the fix but messed up the force push. I had to open a new PR with your changes: #3315.

**Pull request closed without merging**